test/unit/lib/urlutilTest.js

```diff
@@ -562,4 +562,51 @@ describe('urlutil', function () {
       assert.equal(result, 'http://www.test.com/test.pdf')
     })
   })
+
```

Comment:
tests below should be uncommented?

Reply:
Which I guess is why the function passed.

Thanks for catching.